### DISCUSSION TITLE
political content pollute search results

### DISCUSSION THREAD

**Discussion by 54145a** [community/community#61262] [Other Feature Feedback, Questions, & Ideas]
Product Feedback

### Body

This person set up some repos containing political propaganda messages, and polluted the search results by writing a lot of technology-related words in the repo to get users to search for it. I hope the search filters out these phishing content.
https://github.com/cirosantilli

**Comment by moonfury-ops** [comment]
Thank you

**Comment by cirosantilli** [comment]
Unacceptable.

**Comment by LiteBrite82** [comment]
Hi @54145a , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
